content/developers/developer-patterns/getting-access-tokens-using-app-registrations/index.md

```diff
@@ -202,10 +197,39 @@ The header and payload of the `TOKEN` may be examined with the following command
 
 ```shell
 # Header
-echo -n $TOKEN | cut -d '.' -f 1 | base64 -D
+echo -n $TOKEN | cut -d '.' -f 1 | base64 -d | jq
```

Comment:
- `-D` seemed a typo for `-d`
- Added `| jq` to format the output a little cleaner. Although, it does assume the developer has `jq` on their machine.  Not sure if we want to have a doc for common tools that are assumed.

Reply:
We don't have a doc for common tools but jq is mentioned earlier in this article.